site/static/main.css

```diff
@@ -192,3 +192,48 @@ h6 {
     padding: 10px 10px 10px 10px;
   }
 }
+
+.float-right {
+  float: right;
+}
+
+.example {
+  display: flex;
+  flex-direction: column;
+  margin: 24px 0;
+}
+.example.side {
+  flex-direction: row-reverse;
+}
+.example .example-vis {
+  flex-grow: 1;
+  align-self: center;
+  min-width: 300px;
+  text-align: center;
+}
+.example .example-code {
+  width: 550px;
```

Comment:
Lol, already in master. So your call to fix it :-P
On Jan 29, 2016 16:41, "Kanit Wongsuphasawat" <notifications@github.com>
wrote:

> In site/static/main.css
> <https://github.com/vega/vega-lite/pull/1030#discussion_r51334847>:
>
> > +.example {
> > +  display: flex;
> > +  flex-direction: column;
> > +  margin: 24px 0;
> > +}
> > +.example.side {
> > +  flex-direction: row-reverse;
> > +}
> > +.example .example-vis {
> > +  flex-grow: 1;
> > +  align-self: center;
> > +  min-width: 300px;
> > +  text-align: center;
> > +}
> > +.example .example-code {
> > +  width: 550px;
>
> Too bad, should have committed this earlier so you have to fix this for me
> lol
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vega/vega-lite/pull/1030/files#r51334847>.
>